src/lib.rs

```diff
@@ -11,11 +11,15 @@ mod config;
 mod user;
 mod zitadel;
 
-pub use config::{AttributeMapping, Config};
+pub use config::{AttributeMapping, Config, FeatureFlag};
 use zitadel::Zitadel;
 
 /// Run the sync
 pub async fn sync_ldap_users_to_zitadel(config: Config) -> Result<()> {
+	if !config.feature_flags.contains(&FeatureFlag::SsoLogin) {
+		anyhow::bail!("Non-SSO configuration is currently not supported");
```

Comment:
Is it going to be supported at all? Otherwise having this flag is pointless

Reply:
Yes, but we need to wait for zitadel to add support for it. See the commit message.